src/mono/mono/mini/interp/jiterpreter.c

```diff
@@ -1021,8 +1021,28 @@ mono_jiterp_get_options_as_json ()
 	return mono_options_get_as_json ();
 }
 
+EMSCRIPTEN_KEEPALIVE gint32
+mono_jiterp_get_option_as_int (const char *name)
```

Comment:
should this free the buffer ?

Reply:
which buffer?

Reply:
name

Reply:
cwrap/ccall allocate it, so i think they free it afterwards too?

Reply:
ok, you are right `stackAlloc(len)`